### ISSUE TITLE
melpa

### ISSUE THREAD

**Issue by bbigras** [jcs-elpa/codemetrics#2]
melpa please

**Comment by jcs090218** [comment]
Yes, it's on my plan. However, I have other packages in queued so it will take me a while to submit it! ;)

**Comment by jcs090218** [comment]
Oh, BTW. You can install this through [JCS-ELPA](https://github.com/jcs-emacs/jcs-elpa). Most of my packages will publish there first.
